bin/problem.py

```diff
@@ -125,12 +126,23 @@ def _read_settings(self):
             'uuid': None,
             'limits': {},
         }
-        # TODO: implement other limits
         # TODO move timelimit to this?
+        # defaults from https://github.com/Kattis/problem-package-format/blob/master/spec/legacy-icpc.md#limits
         limits = {
             'time_multiplier': 2,
             'time_safety_margin': 1.5,
             'time_resolution': 1.0,
+            'memory': 2048,  # in MiB
```

Comment:
Want to add (some of) these new values (and existing ones) to the skel to make them more visible?
Specifically the time limits ones are useful I guess, but they're already there.

Reply:
I did update the skeleton `problem.yaml` for this, I put `time_limit` there. I was doubting about adding more, but thought it would get very cluttered with things we usually don't need to touch. Of all the `limits`, I think the `time_limit` is the only one we need to change for 90% of the problems, whereas the other settings only need to be changed for less than 10% of the problems (or at least, that's my feeling).

We could link to https://icpc.io/problem-package-format/spec/2023-07-draft.html#limits, but that also feels cluttered to always have in the `problem.yaml` by default... I guess people can google "problem_format_version: 2023-07-draft" and find that webpage as well :stuck_out_tongue:

Reply:
Wouldn't be bad to have a link to the spec in problem.yaml, actually.

Reply:
Fair enough! Linked it in 00cfa1516008144b7c896a5e26530ee371c31fe9 :slightly_smiling_face: The new version of the spec is not fully implemented yet, but it's well underway (with the next step coming up at #421), so I didn't bother to add a disclaimer that it's not complete yet (that also feels cluttery in the skeleton :stuck_out_tongue:)